crates/egui/src/response.rs

```diff
@@ -435,6 +435,18 @@ impl Response {
             }
         }
 
+        if self.ctx.style().interaction.tooltip_delay > 0.0
+            && self.ctx.style().interaction.tooltip_delay
+                > self.ctx.input(|i| i.pointer.time_since_last_movement())
+        {
+            // If a non-zero delay was specified, we only show the tooltip if the mouse has been
+            // still for the specified amount of time. If the mouse is moved, the tooltip
+            // disappears.
+
```

Comment:
As it stands, `tooltip_delay` has double duty:

* Controls delay
* Controls wether or not to close a tooltip by just moving the mouse (when > 0)

Closing tooltips on mouse movement is quite uncommon, and I suggest we never do it (or enable it via its own `bool`).


```suggestion
        if !self.is_tooltip_open()
            && self.ctx.input(|i| i.pointer.time_since_last_movement())
                < self.ctx.style().interaction.tooltip_delay
        {
            // Keep waiting until the mouse has been still for a while
```